src/core/unit-printf.c

```diff
@@ -78,12 +78,18 @@ static int specifier_filename(char specifier, void *data, void *userdata, char *
                 return unit_name_to_path(u->id, ret);
 }
 
+static void bad_specifier(Unit *u, char specifier) {
+        log_unit_warning(u, "Specifier '%%%c' used in unit configuration, which is deprecated. Please update your unit file, as it does not work as intended.", specifier);
```

Comment:
since we don't actually have a replacement shouldn't this read: 'Please remove %%%c from your unit file, as it does not work as intended'? Otherwise we're implying that there isn't a documented replacement.